src/store/slices/node/initialState.ts

```diff
@@ -98,7 +98,13 @@ type InitialState = {
   messagesWebsocketStatus: WebsocketConnectionStatus;
   logsWebsocketStatus: WebsocketConnectionStatus;
   closeChannel: { isFetching: boolean };
-  redeemTickets: { isFetching: boolean };
+  redeemTickets: {
+    isFetching: boolean;
+    error: {
+      status: string | undefined;
+      error: string | undefined;
+    };
```

Comment:
i think we can just have error: string

what do you think @mjadach-iv ?

Reply:
yeah, easier - better